From 899008bb48aa8085f4b06cde8e7a1668da313d8a Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 12 Nov 2019 03:35:31 +0200
Subject: [PATCH] [SW-6379] goya2: add specs project to compiler search path

Now code can include headers from this project
It is needed since we use configuration space registers (Soc on Line)

Change-Id: Iaf35608d93808992ea99c222283d762dc7d7ad61
---

diff --git a/Makefile b/Makefile
index 19eebf9..7dcac13 100644
--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,9 @@
 ARCBOOT_CFLAGS += -Isdk/src/include
 ARCBOOT_CFLAGS += -Isdk/elpsoft/src/headers
 ARCBOOT_CFLAGS += -Isdk/elpsoft/src/math
+SPECS_DEV := $(shell echo $(HL_DEV) | cut -d'_' -f1)
+ARCBOOT_CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(SPECS_DEV)/asic_reg
+ARCBOOT_CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
 
 LDFLAGS = -flto
 LDFLAGS += -L $(LIBDIR) -lsb -lellipsys
